man/generate_man.rb

```diff
@@ -0,0 +1,47 @@
+require 'fileutils'
+def add_header_markup(time)
+  header = ""
+  time.times do
+    header += "-"
+  end
+  header += "\n"
+  header
+end
+
+def trim_colon(string)
+  string.include?(":") ? string.tr(':', '') : string
+end
+
+def available_plugins
+  available = `hammer --version`.split('*')
+  plugins = "Available Plugins\n"
+  plugins << "-----------------\n"
+  available.each do |plugin|
+    plugins << plugin.strip!
+    plugins << "\n\n"
+  end
+  plugins << "\n"
+  plugins
+end
+
+`hammer -h > options.asciidoc`
+tempfile=File.open("file.tmp", 'w')
+f=File.new("options.asciidoc")
+tempfile << available_plugins
+f.each do |line|
+  tempfile << trim_colon(line)
+  if line.downcase=~/^usage:/ || line.downcase=~/^subcommands:/ || line.downcase=~/^parameters:/ || line.downcase=~/^options:/
```

Comment:
For the same reason as above the subcommands will be probably empty so I'd replace it with some hint to list available subcommands with `hammer -h` and that there are man pages for the top-level subcommands e.g. `man hammer-user`